src/Passes/Source/Generators/LlvmPassesConfig.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Commandline/ConfigurationManager.hpp"
+#include "Generators/LlvmPassesConfig.hpp"
+
+namespace microsoft
+{
+namespace quantum
+{
+
+    void LlvmPassesConfiguration::setup(ConfigurationManager& config)
+    {
+        config.setSectionName("LLVM Passes", "Configuration of LLVM passes.");
+        config.addParameter(always_inline_, "always-inline", "Aggresively inline function calls.");
+    }
+
+    LlvmPassesConfiguration LlvmPassesConfiguration::disable()
```

Comment:
It's not clear why this function is called disable and why does it return and object.

Reply:
It is a static function that returns a instance of the configuration which disables all passes.

Reply:
This needs to be documented.

Reply:
Please see #1147.